test/conformance/ingress/util.go

```diff
@@ -341,3 +344,37 @@ func CreateDialContext(t *testing.T, ing *v1alpha1.Ingress, clients *test.Client
 		return nil, errors.New("Service ingress does not contain dialing information.")
 	}
 }
+
+func RuntimeRequest(t *testing.T, client *http.Client, url string) *types.RuntimeInfo {
+	t.Helper()
+
+	// Make a request and check the response.
+	resp, err := client.Get(url)
+	if err != nil {
+		t.Errorf("Error making GET request: %v", err)
+		return nil
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		t.Errorf("Got non-OK status: %d", resp.StatusCode)
+		DumpResponse(t, resp)
+		return nil
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
+	ri := &types.RuntimeInfo{}
+	if err := json.Unmarshal(b, ri); err != nil {
+		t.Errorf("Unable to parse runtime image's response payload: %v", err)
+		return nil
+	}
+	return ri
+}
+
+func DumpResponse(t *testing.T, resp *http.Response) {
+	t.Helper()
+
+	t.Logf("%s %d %s", resp.Proto, resp.StatusCode, resp.Status)
```

Comment:
Dumping both status and status code seem redundant, since `Status=200 oK`.

Reply:
Yeah, it is.  I'd like to get this in so we get more verbose flake tracing, and have a few improvements to make here already. 🤓

Reply:
Also: https://golang.org/pkg/net/http/httputil/#DumpResponse

Reply:
If this flakes again, I'll just push them here, I've got it ready

Reply:
🤔 I wonder if these threats scare our flakes into hiding 🤣